qutebrowser/config/configdata.yml

```diff
@@ -1533,11 +1533,11 @@ url.yank_ignored_parameters:
 
 ## window
 
-window.hide_wayland_decoration:
+window.hide_decoration:
   type: Bool
   default: false
   restart: true
```

Comment:
You should probably remove the `restart: true` after setting it at runtime works, and adjust the description to mention that it requires a restart on wayland (make the description multi-line, see others in this file).